src/unistd.rs

```diff
@@ -735,11 +735,22 @@ pub fn fchown(fd: RawFd, owner: Option<Uid>, group: Option<Gid>) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
-/// Flags for `fchownat` function.
-#[derive(Clone, Copy, Debug)]
-pub enum FchownatFlags {
-    FollowSymlink,
-    NoFollowSymlink,
+#[cfg(not(target_os = "redox"))]
+libc_bitflags! {
+    /// Flags for the `fchownat` function.
+    pub struct FchownatFlags: c_int {
+        /// If  pathname is an empty string, operate on the file referred to
+        /// by `dirfd` (which may have been obtained using the `open(2)`
+        /// `O_PATH` flag). In this case, `dirfd` can refer to any type of
+        /// file, not just a directory.
+        ///
+        /// If `dirfd` is `None`, the call operates on the current working
+        /// directory.
+        #[cfg(linux_android)]
```

Comment:
It's also available on FreeBSD, beginning in 13.1.

Reply:
Let me do this in a separate PR